cmd/truss/main.go

```diff
@@ -56,14 +55,18 @@ func main() {
 	genFiles, err := generateCode(cfg, dt, sd)
 	exitIfError(errors.Wrap(err, "cannot generate service"))
 
-	for _, f := range genFiles {
-		err := writeGenFile(f, cfg.ServicePath)
+	for path, file := range genFiles {
+		err := writeGenFile(file, path, cfg.ServicePath)
 		if err != nil {
 			exitIfError(errors.Wrap(err, "cannot to write output"))
 		}
 	}
 }
 
+// fileSystem is a map of paths relative to truss.Config.ServicePath mapped to
+// io.Reader's as files
```

Comment:
`io.Readers`. Apostrophes are always, always possessive*, never, ever pluralizing. 

*exception: "it's," meaning "it is."

More semantically, the sentence is confusing. "map of X mapped to Y." This is an internal type that's clearly a map of strings to io.Readers. This type is internal, so documentation on what's pretty clear seems kind of unnecessary.

Further, though, this type has no methods attached, so I kind of think it'd make sense to get rid of it and just use `map[string]io.Reader` in its place, which is immediately understandable and doesn't further overload the word "filesystem," which doesn't seem like the most accurate name to me for this type.